### PR TITLE
RS+RMD/YS/RULE19-5 + 19-6

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
I included 19-5 and 6 in the same PR because these are almost the same.

I have one question.
i) I don't think ```UNDETERMINED``` can happen because for example, Case 1 and Case 3 cover all the possible cases thus, Case 5 (undetermined) can't occur. Nonetheless, I included this in the code under comments.

I would appreciate it if you could verify these two issues.

### CODE REVIEW

**Comment by weilixu** [comment]
> I included 19-5 and 6 in the same PR because these are almost the same.
> 
> I have one question. i) I don't think `UNDETERMINED` can happen because for example, Case 1 and Case 3 cover all the possible cases thus, Case 5 (undetermined) can't occur. Nonetheless, I included this in the code under comments.
> 
> I would appreciate it if you could verify these two issues
Rewrite the logic to:
```python
if coincident_unmet_load_hours is not None:
    if coincident_unmet_load_hours <= 300: PASS
    else: FAIL
elif unmet_load_hours_heating is not None and unmet_load_hours_cooling is not None:
    if unmet_load_hours_heating + unmet_load_hours_cooling  <= 300: PASS
    else: FAIL
else:
    UNDETERMINED
```

The UNDETERMINED could be when "unmet_load_hours_heating is None or unmet_load_hours_cooling is None".

**Review by weilixu — APPROVED** [review]
Approved with some comments to address.

File: rct229/rulesets/ashrae9012019/section19/section19rule5.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        unmet_load_hours_cooling_p = output_instance_p["unmet_load_hours_cooling"]
+        coincident_unmet_load_hours_p = output_instance_p["unmet_load_hours"]
+
+        return {
+            "unmet_load_hours_heating_p": unmet_load_hours_heating_p,
+            "unmet_load_hours_cooling_p": unmet_load_hours_cooling_p,
+            "coincident_unmet_load_hours_p": coincident_unmet_load_hours_p,
+        }
+
+    # def manual_check_required(self, context, calc_vals, data=None):
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule5.py]: Addressed.

File: rct229/rulesets/ashrae9012019/section19/section19rule5.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    #     coincident_unmet_load_hours_p = calc_vals["coincident_unmet_load_hours_p"]
+    #
+    #     return not (
+    #         coincident_unmet_load_hours_p <= MAX_COINCIDENT_UNMET_LOAD_HOUR
+    #         or (
+    #             unmet_load_hours_heating_p + unmet_load_hours_cooling_p
+    #             <= MAX_SUM_HEATING_COOLING_UNMET_HOUR
+    #         )
+    #     )
+
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule5.py:56]: I appreciate for clarifying the rule check. Addressed.

File: rct229/rulesets/ashrae9012019/section19/section19rule5.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    def rule_check(self, context, calc_vals=None, data=None):
+        unmet_load_hours_heating_p = calc_vals["unmet_load_hours_heating_p"]
+        unmet_load_hours_cooling_p = calc_vals["unmet_load_hours_cooling_p"]
+        coincident_unmet_load_hours_p = calc_vals["coincident_unmet_load_hours_p"]
+
+        return coincident_unmet_load_hours_p <= MAX_COINCIDENT_UNMET_LOAD_HOUR or (
+            unmet_load_hours_heating_p + unmet_load_hours_cooling_p
+            <= MAX_SUM_HEATING_COOLING_UNMET_HOUR
+        )
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule5.py:69]: Great point! Addressed.

**Pull request merged**